python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -162,7 +179,7 @@ def setUp(self, gangadir=None, wipe_repo=None, extra_opts=[]):
         # This is called before each unittest
         if gangadir is None:
             import os
-            gangadir = os.path.join('$HOME/gangadir_testing', self.__class__.__name__)
+            gangadir = os.path.join('$HOME/gangadir_testing')  # TODO make separate dirs per test suite, `self.__class__.__name__`
```

Comment:
Is this the result of running multiple unittests from the one ```nosetests``` command?
I think in this case this means that the ```tearDownClass``` needs to be expanded to _de-configure_ Ganga once each test is over.
(I think this is a topic for a separate PR but we some tools here ready for when this is addressed :) )

Good spot with regards to that, I never considered the registry would be bleeding due to this :)